File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution{
    public int removeDuplicates(int[] nums){
         int i =0;
         int j = i+1;
         int unique = 1;
         while(i<nums.length-1 && j<nums.length){
            if(nums[j] != nums [j-1]){
                nums[i+1]=nums[j];
                i++;
                j++;
                unique++;
                continue;
            }
            j++;    //i placed at unique elem & only increases when unique elem found.
        }
        return unique;
    }
    
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i=0
        j=0
        n = len(nums)
        for j in range(n):
            if i<1 or nums[j] != nums[i-1]: 
                nums[i] = nums[j]
                i+=1
        return i   